.release-please-manifest.json

```diff
@@ -4,6 +4,7 @@
   "packages/webpack": "0.7.0-beta.0",
   "packages/browserify": "18.1.2",
   "packages/core": "16.2.2",
+  "packages/git-safe-dependencies": "0.0.1",
```

Comment:
we planning to release it as alpha right? was expecting the word `alpha` in there 😛

similar to the `beta` in `"0.7.0-beta.0"`

Reply:
TBF, I'm not enjoying that way of publishing webpack plugin as release-please doesn't handle it for us. these are good for having a new major release be an alpha or RC. version starting with 0 is enough of an indication